src/exoticatechnologies/hullmods/exotics/ExoticHullmodLookup.kt

```diff
@@ -0,0 +1,28 @@
+package exoticatechnologies.hullmods.exotics
+
+import exoticatechnologies.util.AnonymousLogger
+import exoticatechnologies.util.datastructures.Optional
+
+object ExoticHullmodLookup {
+
+    private val lookupMap: HashMap<String, ExoticHullmod> = hashMapOf()
+
+    fun addToLookupMap(exoticHullmod: ExoticHullmod) {
+        lookupMap[exoticHullmod.hullModId] = exoticHullmod
+        AnonymousLogger.log("<-- addToLookupMap()\tadded ExoticHullmod: ${exoticHullmod} to lookup map with ID: ${exoticHullmod.hullModId}\t\tlookupMap: ${lookupMap}", "ExoticHullmodLookup")
```

Comment:
replace with it's internal logger. remove this method. change the other one to logger.error

Reply:
this is probably done. need to verify. yes. done.

Reply:
resolving.